x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -586,6 +586,13 @@ Please see https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-not
 for more details. SQS queue will be configured as a
 https://docs.aws.amazon.com/sns/latest/dg/sns-sqs-as-subscriber.html[subscriber to the SNS topic].
 
+[float]
+=== S3 -> EventBridge -> SQS setup
```

Comment:
Just a thought here: maybe we can add a cloudformation template to help setup s3 -> eventbridge -> sqs? 
We are working on adding a template for the s3-sqs setup: https://github.com/elastic/beats/pull/40642
(not required change for this pr)